### DISCUSSION TITLE
Specifikáció frissítés

### DISCUSSION THREAD

**Discussion by renced42** [nav-gov-hu/Online-Invoice#1041] [Announcements]
Tisztelt Fejlesztők!

Frissítésre került mind a magyar mind az angol nyelvű specifikáció.

Lényegi változás amit már többen is érzékeltek az alábbiak szerint látható:

A módosított dokumentum az Online Számla 3.21.5-ös verziójában – várhatóan 2023. május 15-én – bevezetendő új ERROR-üzenet leírását tartalmazza. Ez az új, blokkoló ERROR-üzenet (INVALID_LINE_OPERATION) tiltja a „MODIFY” értéket a módosító és az érvénytelenítő számlák adatszolgáltatása esetén.
Az új ellenőrzés célja az adatszolgáltatások egységesítése és az adattisztaság növelése, ami az eÁFA-rendszer megfelelő működéséhez is szükséges. 
A számlázóprogramok fejlesztői számára a fenti módosítás tesztelésének lehetősége 2023. február 9-én 16 órától elérhető az Online Számla tesztrendszerben (https://onlineszamla-test.nav.gov.hu/).

**A frissített specifikáció a TESZT rendszerrel van párban. A feltöltött specifikáció a Online Számla tesztrendszerének felületén (https://onlineszamla-test.nav.gov.hu/dokumentaciok) is elérhető.**

**Comment by bmatyi94** [comment]
Kedves @renced42 mi az új dátum amikorra várható az élesítés? Előre is köszönöm a válaszod.

**Reply by NTCA-BA** [reply]
Tisztelt Fejlesztő!
Elnézést kérünk, hogy csak most válaszolunk!
Az Online Számla rendszer új, blokkoló ERROR-üzenetét („INVALID_LINE_OPERATION”) a NAV a korábban tervezett 2023. május 15. helyett szeptember 18-án élesíti. 
További információk ezen a linken találhatók: https://github.com/nav-gov-hu/Online-Invoice/discussions/1067
